backend/src/main/java/com/bakdata/conquery/models/auth/basic/PasswordHasher.java

```diff
@@ -0,0 +1,94 @@
+package com.bakdata.conquery.models.auth.basic;
+
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.security.spec.InvalidKeySpecException;
+
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+
+import com.bakdata.conquery.io.jackson.Jackson;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import jetbrains.exodus.ArrayByteIterable;
+import jetbrains.exodus.ByteIterable;
+import lombok.Data;
+import lombok.experimental.UtilityClass;
+
+/**
+ * Inspired by @see <a href=
+ * "https://stackoverflow.com/questions/18142745/how-do-i-generate-a-salt-in-java-for-salted-hash">https://stackoverflow.com/questions/18142745/how-do-i-generate-a-salt-in-java-for-salted-hash</a>.
+ *
+ */
+@UtilityClass
+public class PasswordHasher {
+
+	private static final SecureRandom RANDOM = new SecureRandom();
+	private static final String ALGORITHM = "PBKDF2WithHmacSHA1";
```

Comment:
Kannst du in einem static block die Parameter (natürlich ohne Random) ins log schreiben?